modules/social_features/social_event/modules/social_event_an_enroll/src/Form/EventAnEnrollActionForm.php

```diff
@@ -110,33 +166,60 @@ public function buildForm(array $form, FormStateInterface $form_state, Node $nod
   }
 
   /**
-   * {@inheritdoc}
+   * Rebuilds form after canceling the event enroll.
+   *
+   * @param array $form
+   *   An associative array containing the structure of the form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The current state of the form.
    */
-  public function submitForm(array &$form, FormStateInterface $form_state) {
-    $uid = $this->currentUser->id();
-
-    $token = $this->getRequest()->query->get('token');
-    if (!empty($token)) {
-      $nid = $form_state->getValue('event');
-      $conditions = [
-        'field_account' => $uid,
-        'field_event' => $nid,
-        'field_token' => $token,
-      ];
+  public function cancelEnrollmentAjax(array &$form, FormStateInterface $form_state): AjaxResponse {
+    $response = new AjaxResponse();
+    $token = '';
```

Comment:
[Here](https://github.com/goalgorilla/open_social/pull/2830/files#diff-3224e902e48f436226971baade8cec71c84c9fdf4d1ceef97f251f7873be3049R149), default token = NULL, should in this case be also NULL?